inc/formanswer.class.php

```diff
@@ -744,6 +744,12 @@ public function prepareInputForUpdate($input) {
             ];
             $skipValidation = true;
          } else {
+            //If you are accepting an answer, also limit it to just these 3 fields.
+            $input = [
+               'id'      => $input['id'],
+               'status'  => $input['status'],
+               'comment' => isset($input['comment']) ? $input['comment'] : 'NULL',
+            ];
             // Check if a field is being edited
             $fieldPresence = [];
             $this->getQuestionFields($input['plugin_formcreator_forms_id']);
```

Comment:
With your modification, this line will trigger a PHP warning and al question objects will fail to load

Reply:
try this PR and feedback : https://github.com/pluginsGLPI/formcreator/pull/2398